src/integTest/java/info/blockchain/wallet/BaseIntegTest.java

```diff
@@ -45,7 +45,12 @@ public Retrofit getRetrofitServerInstance() {
 
             @Override
             public Retrofit getRetrofitSFOXInstance() {
-                return null;
+                return getRetrofit(PersistentUrls.getInstance().getCurrentSFOXUrl(), getOkHttpClient());
+            }
+
+            @Override
+            public Retrofit getRetrofitCoinifyInstance() {
+                return getRetrofit(PersistentUrls.getInstance().getCurrentCoinifyUrl(), getOkHttpClient());
```

Comment:
I think we should change how this works in the future; the number of necessary Retrofit instances is getting a bit silly now.